Description: Add Kubuntu XML feeds to Akregrator defaults
 This possibly should be upstreamed by implementing a mechanism to enable
 distros to easily do this without having to patch the source. Load a simple
 json file maybe?
Author: Harald Sitter <sitter@kde.org>
Origin: vendor
Forwarded: not
Last-Update: 2016-03-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git kdepim/akregator/src/akregator_part.cpp kdepim/akregator/src/akregator_part.cpp
index bf6a913..cb39196 100644
--- kdepim/akregator/src/akregator_part.cpp
+++ kdepim/akregator/src/akregator_part.cpp
@@ -176,6 +176,30 @@ static QDomDocument createDefaultFeedList()
     spanishKde.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://planet.kde-espana.org/atom.xml"));
     spanishFolder.appendChild(spanishKde);

+    QDomElement kubuntuFolder = doc.createElement(QStringLiteral("outline"));
+    kubuntuFolder.setAttribute(QStringLiteral("text"), QStringLiteral("Kubuntu"));
+    body.appendChild(kubuntuFolder);
+
+    QDomElement planet = doc.createElement(QStringLiteral("outline"));
+    planet.setAttribute(QStringLiteral("text"), i18n("Planet Ubuntu"));
+    planet.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://planet.ubuntu.com/rss20.xml"));
+    kubuntuFolder.appendChild(planet);
+
+    QDomElement fridge = doc.createElement(QStringLiteral("outline"));
+    fridge.setAttribute(QStringLiteral("text"), i18n("Ubuntu Fridge"));
+    fridge.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://feeds.feedburner.com/ubuntu-fridge"));
+    kubuntuFolder.appendChild(fridge);
+
+    QDomElement ktimes = doc.createElement(QStringLiteral("outline"));
+    ktimes.setAttribute(QStringLiteral("text"), i18n("Kubuntu.org News"));
+    ktimes.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://www.kubuntu.org/news/feed"));
+    kubuntuFolder.appendChild(ktimes);
+
+    QDomElement wire = doc.createElement(QStringLiteral("outline"));
+    wire.setAttribute(QStringLiteral("text"), i18n("Kubuntu Wire"));
+    wire.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://wire.kubuntu.org/?feed=rss2"));
+    kubuntuFolder.appendChild(wire);
+
     return doc;
 }
 }
